
package com.laegler.openbanking.soap.service;

import javax.xml.ws.WebFault;


/**
 * Not Acceptable
 *
 * This class was generated by Apache CXF 3.3.3
 * 2019-10-21T07:01:26.897+13:00
 * Generated source version: 3.3.3
 */

@WebFault(name = "PUT_ChangeEventSubscriptionsEventSubscriptionId_406", targetNamespace = "http://laegler.com/openbanking/soap/model")
public class PUTChangeEventSubscriptionsEventSubscriptionId406 extends Exception {

    private java.lang.Object putChangeEventSubscriptionsEventSubscriptionId406;

    public PUTChangeEventSubscriptionsEventSubscriptionId406() {
        super();
    }

    public PUTChangeEventSubscriptionsEventSubscriptionId406(String message) {
        super(message);
    }

    public PUTChangeEventSubscriptionsEventSubscriptionId406(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public PUTChangeEventSubscriptionsEventSubscriptionId406(String message, java.lang.Object putChangeEventSubscriptionsEventSubscriptionId406) {
        super(message);
        this.putChangeEventSubscriptionsEventSubscriptionId406 = putChangeEventSubscriptionsEventSubscriptionId406;
    }

    public PUTChangeEventSubscriptionsEventSubscriptionId406(String message, java.lang.Object putChangeEventSubscriptionsEventSubscriptionId406, java.lang.Throwable cause) {
        super(message, cause);
        this.putChangeEventSubscriptionsEventSubscriptionId406 = putChangeEventSubscriptionsEventSubscriptionId406;
    }

    public java.lang.Object getFaultInfo() {
        return this.putChangeEventSubscriptionsEventSubscriptionId406;
    }
}
